tests/ui/perfherder/graphs-view/graphs_view_test.jsx

```diff
@@ -216,6 +221,42 @@ test('Using select query param displays tooltip for correct datapoint', async ()
   expect(platform).toHaveTextContent(testData[0].platform);
 });
 
+test("Alert's ID can be copied to clipboard from tooltip", async () => {
+  const selectedDataPoint = {
+    signature_id: testData[0].signature_id,
+    dataPointId: testData[0].data[5].id,
+  };
+
+  Object.assign(navigator, {
+    clipboard: {
+      writeText: jest.fn(),
+    },
+  });
+  const { getByTestId, queryAllByTitle } = graphsViewControls(
+    graphData,
+    false,
+    undefined,
+    undefined,
+    selectedDataPoint,
+  );
+
+  const graphContainer = await waitFor(() => getByTestId('graphContainer'));
+  expect(graphContainer).toBeInTheDocument();
+
+  const graphTooltip = await waitFor(() => getByTestId('graphTooltip'));
+  expect(graphTooltip).toBeInTheDocument();
+
+  const copyIdButtons = await waitFor(() =>
+    queryAllByTitle('Copy Alert Summary id'),
+  );
```

Comment:
```suggestion
  const copyIdButton = await waitFor(() =>
    queryByTitle('Copy Alert Summary id'),
  );
```
If you have only one data point in the graph then I think this could work, no need to query all if you have one single item. Let me know what you think. 
@esanuandra

Reply:
You change it to `queryByTitle`, but it will throw an error if more data points are added later on.
For our case `queryByTitle` is just enough.